src/snap/metric.cc

```diff
@@ -15,10 +15,12 @@ limitations under the License.
 
 #include <utility>
 #include <chrono>
+#include <cstdint>
```

Comment:
The header file with the declaration of these functions already has this include, I don't see a need to put it here as well. It's the same with the other includes.